EasyLearn/app/views/login/index.html.erb

```diff
@@ -1,35 +1,35 @@
 
-<%= stylesheet_link_tag "login", "data-turbo-track": "reload" %>
-<body>
+
+
 	<section class="vh-100 gradient-custom">
 		<div class="container h-100">
 			<div class="row d-flex justify-content-center align-items-center h-100">
 				<div class="col-12 col-md-8 col-lg-6 col-xl-5">
```

Comment:
Check the indentation of your file.

Reply:
The indentation is correct, we have maybe  because we have a lot of divs, but the indentation is correct.

Reply:
Done.